pandas/core/series.py

```diff
@@ -2576,10 +2576,136 @@ def _needs_reindex_multi(self, axes, method, level):
         """
         return False
 
-    @Appender(generic._shared_docs['align'] % _shared_doc_kwargs)
     def align(self, other, join='outer', axis=None, level=None, copy=True,
               fill_value=None, method=None, limit=None, fill_axis=0,
               broadcast_axis=None):
+        """
+        Align two Series on their axes with the specified join method for each
+        axis Index.
+
+        The result is a tuple containing two Series with the same indexes.
```

Comment:
tuple of (Series, type of other)

Reply:
I added a line break, such that `join` and the info about MultiIndex are separate.
```
The result is a tuple of (Series, type of other) with the same indexes.

Using `join`, one can specifiy if the union or intersection of the
indexes are retained or whether to keep the index of the first or
second Series.
```